hw/ip/ascon/data/ascon.hjson

```diff
@@ -0,0 +1,696 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+# Ascon register template
+{
+  name: "ascon",
+  regwidth: "32",
+  cip_id: "40",
+  version: "0.0.1",
+  clocking: [
+    {clock: "clk_i", reset: "rst_ni", idle: "idle_o", primary: true},
+    {clock: "clk_edn_i", reset: "rst_edn_ni", idle: "idle_o"}
+    ],
+  bus_interfaces: [
+    { protocol: "tlul", direction: "device" }
+  ],
+
+  alert_list: [
+    { name: "recov_ctrl_update_err",
+      desc: '''
+        This recoverable alert is triggered upon detecting an update error in the shadowed Control Register.
+        The content of the Control Register is not modified (See Control Register).
+        The Ascon unit can be recovered from such a condition by restarting the Ascon operation, i.e., by re-writing the Control Register.
+        This should be monitored by the system.
+      '''
+    }
+    { name: "fatal_fault",
+      desc: '''
+        This fatal alert is triggered upon detecting a fatal fault inside the Ascon unit.
+        Examples for such faults include
+        i) storage errors in the shadowed Control Register,
+        ii) any internal FSM entering an invalid state,
+        iii) any sparsely encoded signal taking on an invalid value,
+        iv) errors in the internal round counter,
+        v) escalations triggered by the life cycle controller, and
+        vi) fatal integrity failures on the TL-UL bus.
```

Comment:
bus integrity error needs to be ORed to the common fatal alert.

Reply:
This refers to `assign unused_err_alert = intg_err_alert;` and not to the hjson itself, right?